test/builtin/test_makeboxes.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+import pytest
+from test.helper import check_evaluation, session
+from mathics_scanner.errors import IncompleteSyntaxError
+
+# 15 tests
+@pytest.mark.parametrize(
+    ("str_expr", "str_expected", "msg"),
+    [
+        (r"MakeBoxes[1.4]", r"1.4`", None),
+        (r"MakeBoxes[1.4`]", r"1.4`", None),
+        (r"MakeBoxes[1.5`20]", r"1.5`20.", None),
+        (r"MakeBoxes[1.4`20]", r"1.4`20.", None),
+        (r"MakeBoxes[1.5``20]", r"1.5`20.1760912591", None),
+        (r"MakeBoxes[-1.4]", r'RowBox[{"-", 1.4`}]', None),
+        (r"MakeBoxes[34.*^3]", r"34000.`", None),
+        (r"MakeBoxes[0`]", r"0.`", None),
+        (r"MakeBoxes[0`3]", r"0", None),
+        (r"MakeBoxes[0``30]", r"0.``30.", None),
+        (r"MakeBoxes[0.`]", r"0.`", None),
+        (r"MakeBoxes[0.`3]", r"0.`", None),
+        (r"MakeBoxes[0.``30]", r"0.``30.", None),
+        (r"MakeBoxes[14]", r"14", None),
+        (r"MakeBoxes[-14]", r'RowBox[{"-", 14}]', None),
+    ],
+)
+@pytest.mark.xfail
```

Comment:
This is a skipped test. xfail is for testing properties that  _properly_ fail. Like divide by zero. Not things that currently happen to be bugs.k Thanks.

Reply:
@mmatera I am wrong. I checked https://docs.pytest.org/en/7.1.x/how-to/skipping.html and it does mention bugs. 

(That said, my own use of of skip is for bugs not expected failures.

Reply:
@rocky, I used `xfail` just to have a notion about the progress. However, it is true that it takes time to check that something still does not work. So, now I make it optional and controlled by the "DEBUG" environment variable.

Reply:
BTW, thanks for the reference. I got some ideas to improve the implementation  of some of the tests.